helm/policies-aws/values.yaml

```diff
@@ -0,0 +1,7 @@
+region: "eu-west-1"
```

Comment:
Has there been any broader discussion/decision about where defaults should be defined? I was thinking it would be beneficial to have them in a CR in the cluster so that they are also accessible to UI tools and other projects which might want to read them. I also think it will be important to be able to determine with which version of defaults a given cluster was created.

Reply:
We talked about this before and the current thinking was to utilize github.com/giantswarm/config for the definition of default values. Making them accessible would then happen through a configmap ideally.

Not sure if this is how we are going to keep it but to get things moving a CM should be sufficient.